### PR TITLE
[clickhouse/exporter] Fix clickhouse exporter insert metrics data bug

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
As the insert SQL show:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/66acad234166a2f59347c01b44837a0ca51354e1/exporter/clickhouseexporter/internal/gauge_metrics.go#L70-L77

`traceIDs` and `spanIDs` are in error order in  `ExecContext` function:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/66acad234166a2f59347c01b44837a0ca51354e1/exporter/clickhouseexporter/internal/gauge_metrics.go#L129-L133

### CODE REVIEW

File: exporter/clickhouseexporter/exporter_metrics_test.go [hunk 1]
```diff
@@ -97,6 +99,30 @@ func TestExporter_pushMetricsData(t *testing.T) {
...
+			var sumMetrics = regexp.MustCompile(`^INSERT INTO otel_metrics_sum `)
+			if strings.HasPrefix(query, "INSERT INTO otel_metrics_gauge") {
+				require.Equal(t, clickhouse.ArraySet{"0102030000000000"}, values[18])
+				require.Equal(t, clickhouse.ArraySet{"01020300000000000000000000000000"}, values[19])
+			}
+			if strings.HasPrefix(query, "INSERT INTO otel_metrics_histogram") {
+				require.Equal(t, clickhouse.ArraySet{"0102030000000000"}, values[20])
+				require.Equal(t, clickhouse.ArraySet{"01020300000000000000000000000000"}, values[21])
+			}
+			if sumMetrics.MatchString(query) {
```
Comment (bogdandrutu) [inline exporter/clickhouseexporter/exporter_metrics_test.go:113]: Why not using HasPrefix here?

**Review by bogdandrutu — COMMENTED** [review]
Where can we read about the documentation or ordering?

File: exporter/clickhouseexporter/exporter_metrics_test.go (hunk 1 above)
Comment (Frapschen) [reply exporter/clickhouseexporter/exporter_metrics_test.go:113]: I had tried HasPrefix but it also will match `otel_metrics_summary`

**Comment by Frapschen** [comment]
> Where can we read about the documentation or ordering?

I updated the Description, you can see it now

**Comment by Frapschen** [comment]
@bogdandrutu Please review it

File: exporter/clickhouseexporter/exporter_metrics_test.go (hunk 1 above)
Comment (sfc-gh-bdrutu) [reply exporter/clickhouseexporter/exporter_metrics_test.go:113]: It is hard to believe that `strings.HasPrefix(query, "INSERT INTO otel_metrics_sum ")` matches the summary. Did you by mistake remove the space after `m`?

**Review by djaglowski — APPROVED** [review]

**Pull request merged**